tests/handlers_test.go

```diff
@@ -0,0 +1,34 @@
+package handlers
+
+import (
+	"net/http"
+	"testing"
+	"web/src/config"
+	"web/src/handlers"
+	"web/src/model"
+
+	"github.com/go-playground/assert/v2"
+	"github.com/jarcoal/httpmock"
+)
+
+func TestReviewServiceStatusWhenAPIDown(tests *testing.T) {
+	apiPort, apiPath, _ := config.ConfigureEnvironmentals()
+
+	// Test case for when the API is down
+	httpmock.Activate()
+	defer httpmock.DeactivateAndReset()
+
+	httpmock.RegisterResponder("GET", "http://localhost:"+apiPort+apiPath, httpmock.NewErrorResponder(http.ErrServerClosed))
+	services := []model.ServiceInfo{
+		{Name: "UI", URL: "http://localhost:" + apiPort},
+		{Name: "API", URL: "http://localhost:" + apiPort + apiPath},
+	}
```

Comment:
```suggestion
	services := []model.ServiceInfo{
		{Name: "UI", URL: "http://"  + appHost + ":" + apiPort},
		{Name: "API", URL: "http://" + appHost + ":" + apiPort + apiPath},
	}
```

And also add here check, if FORCE_HTTPS = true, then use https instead of http